### ISSUE TITLE
Fresh Install ver 4.0: 500 Internal server error

### ISSUE THREAD

**Issue by radhikavm** [akeneo/pim-community-dev#12525]
Referring to the following documentation: 
[https://docs.akeneo.com/4.0/install_pim/manual/index.html ](https://docs.akeneo.com/4.0/install_pim/manual/index.html)
always tend to get the following error: using Ubuntu installation:

![Screen Shot 2020-07-30 at 10 50 16 pm](https://user-images.githubusercontent.com/1100207/88932066-effb5080-d2c0-11ea-8721-4bfa38ba12ea.png)

There are no errors when I run `NO_DOCKER=true make prod` . 
Checked error logs on apache as well as `/var/logs/` nothing showing 

Some help with this will be highly appreciated.

**Comment by codeagencybe** [comment]
I have exactly the same problem. 
Any solution found yet?

**Comment by radhikavm** [comment]
Unfortunately non thus far..

**Comment by astrashilov** [comment]
Check the DB - I had the same problem and there was missing "pim_session" table in MySQL DB. After I have recreated it from another running DB it turned out that the DB creation and import of data was broken and the other tables which were successfully created were actually empty.

**Comment by dalarcia** [comment]
Hi.

¿Have you checked permissions on folders public/, and var/ of the pim folder? I'm used to get this error every time I clear akeneo cache because the owner of these folder changes to root user.

Best regards.

**Comment by radhikavm** [comment]
Yes I did check the permissions on the directories mentioned above. What is really difficult is that I see no errors after I go through the installation as when I run all the commands as per the documentation I can see all assets created as required and the frontend getting built as expected as well. But yet get the internal server error as above, Nothing in the logs as well !)

**Comment by ildemartinez** [comment]
I solved it as @radhikavm commented with -> chown -R www-data:www-data akeneo/

**Comment by sureshtakeda** [comment]
Hi Guys,

On PIM 7 with the new installation, I'm getting the above error. Could someone provide any solution to this?

**Comment by devendra-nulogic** [comment]
In var/logs/prod.log file below error i got:
Base table or view not found: 1146 Table 'akeneo_pim.pim_session' doesn't exist

To resolve this:

First check is ElasticSearch is enabled,
`sudo service elasticsearch status`

if its enabled and working then execute below cammands:
`NO_DOCKER=true make prod`

Now check the database Table should be created
Set the 777 permission to application: issue should be resolved.

![pim_session](https://github.com/akeneo/pim-community-dev/assets/142531771/52dce0c6-134a-49a2-9866-99c23cebe0a3)
